docs/sources/reference/api.md

```diff
@@ -744,210 +937,88 @@ Services can have the following states:
-- `end=<nanosecond Unix epoch>`: End timestamp.
-- `since`: A `duration` used to calculate `start` relative to `end`. If `end` is in the future, `start` is calculated as this duration before now. Any value specified for `start` supersedes this parameter.
-
-You can URL-encode these parameters directly in the request body by using the POST method and `Content-Type: application/x-www-form-urlencoded` header. This is useful when specifying a large or dynamic number of stream selectors that may breach server-side URL character limits.
-
-In microservices mode, these endpoints are exposed by the querier.
-
-### Examples
-
-```console
-$ curl -s "http://localhost:3100/loki/api/v1/series" --data-urlencode 'match[]={container_name=~"prometheus.*", component="server"}' --data-urlencode 'match[]={app="loki"}' | jq '.'
-{
-  "status": "success",
-  "data": [
-    {
-      "container_name": "loki",
-      "app": "loki",
-      "stream": "stderr",
-      "filename": "/var/log/pods/default_loki-stack-0_50835643-1df0-11ea-ba79-025000000001/loki/0.log",
-      "name": "loki",
-      "job": "default/loki",
-      "controller_revision_hash": "loki-stack-757479754d",
-      "statefulset_kubernetes_io_pod_name": "loki-stack-0",
-      "release": "loki-stack",
-      "namespace": "default",
-      "instance": "loki-stack-0"
-    },
-    {
-      "chart": "prometheus-9.3.3",
-      "container_name": "prometheus-server-configmap-reload",
-      "filename": "/var/log/pods/default_loki-stack-prometheus-server-696cc9ddff-87lmq_507b1db4-1df0-11ea-ba79-025000000001/prometheus-server-configmap-reload/0.log",
-      "instance": "loki-stack-prometheus-server-696cc9ddff-87lmq",
-      "pod_template_hash": "696cc9ddff",
-      "app": "prometheus",
-      "component": "server",
-      "heritage": "Tiller",
-      "job": "default/prometheus",
-      "namespace": "default",
-      "release": "loki-stack",
-      "stream": "stderr"
-    },
-    {
-      "app": "prometheus",
-      "component": "server",
-      "filename": "/var/log/pods/default_loki-stack-prometheus-server-696cc9ddff-87lmq_507b1db4-1df0-11ea-ba79-025000000001/prometheus-server/0.log",
-      "release": "loki-stack",
-      "namespace": "default",
-      "pod_template_hash": "696cc9ddff",
-      "stream": "stderr",
-      "chart": "prometheus-9.3.3",
-      "container_name": "prometheus-server",
-      "heritage": "Tiller",
-      "instance": "loki-stack-prometheus-server-696cc9ddff-87lmq",
-      "job": "default/prometheus"
-    }
-  ]
-}
+GET, POST, DELETE /ingester/prepare_shutdown
 ```
 
-## Index Stats
+This endpoint is used to tell the ingester to release all resources on next SIGTERM.
 
-The `/loki/api/v1/index/stats` endpoint can be used to query the index for the number of `streams`, `chunks`, `entries`, and `bytes` that a query resolves to.
+After a `POST` to the `prepare_shutdown` endpoint returns, when the ingester process is stopped with `SIGINT` / `SIGTERM`,
```

Comment:
After a `POST` to [are we missing a word here?] the `prepare_shutdown` endpoint returns....